recipes/libgit2/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
   "0.28.3":
     folder: all
+  "0.27.9":
+    folder: 0.27.x
```

Comment:
If you specifically need `0.27.9`, it's ok; but please add 0.27.10 too as it is the last one of that series.